openupgradelib/openupgrade.py

```diff
@@ -110,6 +110,14 @@ def do_raise(error):
 else:
     api = False
 
+# Setting the target version as an environment variable allows OpenUpgrade
+# to skip methods that are called in every version but really only need to
+# run in the target version. Make the target version available to OpenUpgrade
+# with `export OPENUPGRADE_TARGET_VERSION=13.0` (when migrating up to 13.0)
```

Comment:
I'd like to see this paragraph somewhere in the docs too

Reply:
Thanks for suggesting! I have now created https://github.com/OCA/OpenUpgrade/pull/2619 for that.